src/main/java/dev/openfeature/sdk/MutableContext.java

```diff
@@ -88,7 +90,7 @@ public MutableContext add(String key, List<Value> value) {
     @Override
     public EvaluationContext merge(EvaluationContext overridingContext) {
         if (overridingContext == null) {
-            return new MutableContext(this.asMap());
+            return new MutableContext(this.targetingKey, this.asMap());
```

Comment:
Is this a bugfix?

Reply:
Yes. I was increasing test coverage and found this bug. :-/

Reply:
Good fix. I think the only thing to be aware of is that if we fix it in this PR, it won't be captured in the release notes (release please generates the notes based on semantic commits).